### PR TITLE
drainer: update read-timeout comment

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tidb-binlog/issues/1169

### What is changed and how it works?
Add read-timeout and its explanations on drainer's toml config.

### Check List 

Tests 

- None

Code changes

- None

Side effects

- None

Related changes

- None

### Release note

- No release note

### CODE REVIEW

File: cmd/drainer/drainer.toml [hunk 1]
```diff
@@ -104,6 +104,10 @@ password = ""
 # if encrypted_password is not empty, password will be ignored.
 encrypted_password = ""
 port = 3306
+# I/O read timeout for drainer's downstream db connection.
+# default value is 1m, if drainer keeps failing on some long-time costing ddls, you can set this variable to a larger time
```
Comment (shichun-0415) [inline cmd/drainer/drainer.toml:108]:
```suggestion
# I/O read timeout of database connections downstream.
# The default value is `"1m"`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```

File: cmd/drainer/drainer.toml [hunk 2]
```diff
@@ -104,6 +104,10 @@ password = ""
...
+# read-timeout = "10m"
```
[the remark this answers is not in the document]
Comment (lichunzhu) [reply cmd/drainer/drainer.toml:109]:
```suggestion
# read-timeout = "1m"
```

**Review by shichun-0415 — APPROVED** [review]

File: cmd/drainer/drainer.toml [hunk 3]
```diff
@@ -104,6 +104,10 @@ password = ""
 # if encrypted_password is not empty, password will be ignored.
 encrypted_password = ""
 port = 3306
+# I/O read timeout of database connections downstream.
+# The default value is `"1m"`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```
Comment (hi-rustin) [inline cmd/drainer/drainer.toml:108]:
```suggestion
# The default value is `1m`. If Drainer keeps failing on some DDLs that take a long time, you can set this configuration to a larger value.
```

Does this change make sense?

**Review by hi-rustin — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**